src/main/resources/META-INF/services/com.mycila.maven.plugin.license.PropertiesProvider

```diff
@@ -0,0 +1 @@
+org.hawkular.build.license.CopyrightRangeProvider
```

Comment:
Minor: missing new line. You need the Hawkular tools to check Hawkular tools source ;)